styles/style.css

```diff
@@ -104,4 +104,8 @@
   text-decoration-line: underline;
   text-align: left;
   margin-left: 10px;
+}
+
+.two-column {
+  width: 50%;
 }
```

Comment:
It seems like the same issue within https://github.com/reviakin/gorkogo55-landing/issues/72 Please read it there. 
Nevertheless if you will note previous comment you don't need to do anything in this file.